### ISSUE TITLE
[Q]How can I show the loss gap in figure?

### ISSUE THREAD

**Issue by pierowu** [wandb/wandb#8128]
![image](https://github.com/user-attachments/assets/2d71e3c9-6ab6-4851-97fc-4f34c7917f8d)
This is my loss figure. I wanna show the gap between the loss in first step and the loss in last step. How can I realize this goal? Thanks.

**Comment by luisbergua** [comment]
Hey @pierowu, thanks for writing in! Would you mind sharing what's your use case so we can provide the available options?

**Comment by pierowu** [comment]
> Hey @pierowu, thanks for writing in! Would you mind sharing what's your use case so we can provide the available options?

Thanks for reply. I'm finetuning a LLM. Wandb is used to track the loss function. And I wanna show the gap between the initial loss value and the last loss value, so that I can see if the loss decline significantly.
I want to achieve an effect similar to the red text in the picture.
![image](https://github.com/user-attachments/assets/7322d381-52b5-4f1e-ab7e-e9139f31919f)

**Comment by fmamberti-wandb** [comment]
HI @pierowu, this is currently not possible however, I have logged in a Feature Request for our product team to review and we will keep you posted with any updates on this.
